content/get-involved/index.js

```diff
@@ -8,38 +8,50 @@ const meta = {
 }
 
 const hero = {
-    preheading: {
-        en: 'Get Involved',
-    },
     heading: {
-        en: 'How to Help with STAC',
+        en: 'Get Involved',
     },
     body: {
         en: `Welcome! The STAC community prides itself on being welcoming and collaborative, and we are excited you clicked on this page to learn how you might help!
 
 If you aren't sure where to begin, join the [STAC Community Google Group](https://groups.google.com/a/cloudnativegeo.org/g/stac-community).
-Upon signup, you will receive invitations to the biweekly STAC Community meetup (every other Monday at 11 am EST).  [Use this time zone converter if needed](https://www.timeanddate.com/worldclock/converter.html?iso=20231121T160000&p1=224&p2=75&p3=64&p4=263&p5=168).
+This will get you invites to the community meetings (see below).
 
-Looking for online discussions and activity around STAC? Check out the [stac-spec repo](https://github.com/radiantearth/stac-spec), [stac-api-spec repo](https://github.com/radiantearth/stac-api-spec), and the [stac-utils repo](https://github.com/stac-utils) to see
-discussions and updates to STAC and its ecosystem. We are also have a [STAC category in the CNG Discourse](https://cng.discourse.group/c/stac)`,
+We also invite you to join the [CNG Slack](https://join.slack.com/t/cloudnativegeo/shared_invite/zt-2i37pc3nm-9bXiRHYrI6fH5qFh2VaLFA), there's a #stac channel where you can ask questions and get help from the community.
+
+Lastly, for bigger discussions and more in-depth technical conversations, we have a [discussion forum on GitHub](https://github.com/radiantearth/stac-spec/discussions). There are also the the [stac-spec](https://github.com/radiantearth/stac-spec), [stac-api-spec](https://github.com/radiantearth/stac-api-spec), and the [stac-utils](https://github.com/stac-utils) GitHub repositories where you can write issues, feature requests, propose changes, or contribute to the specification and tools.`
```

Comment:
```suggestion
Lastly, for bigger discussions and more in-depth technical conversations, we have a [discussion forum on GitHub](https://github.com/radiantearth/stac-spec/discussions). For specific issues, feature requests, and to propose changes to the specification or tooling please open an issue directly on the relevant GitHub repository: [stac-spec](https://github.com/radiantearth/stac-spec), [stac-api-spec](https://github.com/radiantearth/stac-api-spec), or [stac-utils](https://github.com/stac-utils).
```

Reply:
This changes the initial meaning though, I meant to refer to issues, PRs, discussions and code editing, not just to open issues for everything.

A proposal that mixes both:

```suggestion
Lastly, for bigger discussions and more in-depth technical conversations, we have a [discussion forum on GitHub](https://github.com/radiantearth/stac-spec/discussions). To report issues, request features, propose changes, or contribute to the specification and tools feel encouraged to directly head to the [stac-spec](https://github.com/radiantearth/stac-spec), [stac-api-spec](https://github.com/radiantearth/stac-api-spec), and the [stac-utils](https://github.com/stac-utils) GitHub repositories.`
```

Reply:
Love it!